config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "application",
+    "password": "Password123",
+    "database": "petswap",
+    "host": "127.0.0.1",
```

Comment:
Replace this with your local mysql ip address. Chances are, this can stay as it is.